hw05-framework/src/main/java/com/remifo/TestFramework.java

```diff
@@ -0,0 +1,88 @@
+package com.remifo;
+
+import com.remifo.annotations.After;
+import com.remifo.annotations.Before;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.HashSet;
+import java.util.Set;
+
+public class TestFramework {
+
+    private int numberSuccessfulTest;
+    private int numberUnsuccessfulTest;
+
+    public void startTest( String name ) {
```

Comment:
Не понятно, что за name
```suggestion
    public void startTest( String className ) {
```